x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.tsx

```diff
@@ -223,7 +224,7 @@ export const RuleDetails: React.FunctionComponent<RuleDetailsProps> = ({
                 </EuiFlexGroup>
               )}
             </>,
-            { i18n: i18nStart, theme }
+            { i18n: i18nStart, theme, userProfile }
```

Comment:
Not passing a large object in as the service dependencies here, because this is within a `useEffect`. See https://github.com/elastic/kibana/pull/202606#discussion_r1871588439